ipaserver/install/httpinstance.py

```diff
@@ -429,7 +426,14 @@ def configure_mod_ssl_certs(self):
                                    paths.IPA_CA_CRT, False)
 
     def __publish_ca_cert(self):
-        self.export_ca_certs_file(paths.CA_CRT, self.ca_is_configured)
+        ca_subject = self.cert.issuer
+        certlist = x509.load_certificate_list_from_file(paths.IPA_CA_CRT)
+        ca_certs = [c for c in certlist if c.subject == ca_subject]
+        if not ca_certs:
+            raise RuntimeError("HTTPD cert was issued by an unknown CA.")
+        # at this time we can assume any CA cert will be valid since this is
+        # only run during installation
+        x509.write_certificate(ca_certs[0], paths.CA_CRT)
```

Comment:
Just curious why you went with a list here instead of a single certificate object? If there is more than one matching subject should things blow up anyway?

Reply:
I am not sure there could be a case where there would be multiple certificates with the same subject in `paths.IPA_CA_CRT`. I assumed there would be only one, but the easiest way to get it from list of CA certs in `paths.IPA_CA_CRT` was a list comprehension which still results in a list. If you want, I could perhaps add a check that there's just one cert in the filtered-out list.

Reply:
Multiple subjects with the same subject and SKID are a valid use case. For example they are used to provide an alternative chain to an alternative root CA. For example Let's Encrypt is still cross signed by two CAs.

## letsencryptauthorityx3.pem
```
Issuer: C = US, O = Internet Security Research Group, CN = ISRG         
Subject: C = US, O = Let's Encrypt, CN = Let's Encrypt Authority X3
```

## lets-encrypt-x3-cross-signed.pem
```
Issuer: O = Digital Signature Trust Co., CN = DST Root CA X3
Subject: C = US, O = Let's Encrypt, CN = Let's Encrypt Authority X3
```